confgenerator/config.go

```diff
@@ -127,6 +128,17 @@ func newValidator() *validator.Validate {
 		}
 		return t >= tmin
 	})
+	// url validates that the value is a valid url
+	v.RegisterValidation("url", func(fl validator.FieldLevel) bool {
```

Comment:
We could consider having `http_url`, `jmx_url`, etc, so that some more specific validation could be done.

Reply:
This is my first encounter with the `validator` library, so I just wasn't aware that it contained built in validators. It looks to do just what we want, so I'll switch to that.